src/zimmount/zimmount.h

```diff
@@ -0,0 +1,38 @@
+#ifndef OPENZIM_ZIMMOUNT_H
```

Comment:
Do we really need a `zimmount.h` ?
`zimmount` is the main binary, it cannot be "included" by other program.

But we can create a `zim_node.h`/`zim_node.cpp` as it will probably grow up with time.